ansible/roles/foreman_proxy/templates/tftp.yml.j2

```diff
@@ -8,3 +8,7 @@
 
 # Defines the TFTP Servername to use, overrides the name in the subnet declaration
 #:tftp_servername: tftp.domain.com
+
+{% if foreman_proxy_tftp_host is defined %}
+:tftp_servername: {{ foreman_proxy_tftp_host }}
```

Comment:
Should we change the variable name to `foreman_proxy_tftp_servername` thus it's more aligned with the corresponding Foreman setting?

Reply:
I agree that foreman_proxy_tftp_host was a poor choice.  I could see using foreman_proxy_tftp_servername but I am of the opinion that the variables should be more consistent across ansible than sync'd with the application ansible is configuring.  I see foreman_proxy_dhcp_server is already used so I could see changing it to foreman_proxy_tftp_server also.  Do you have a strong preference?

Reply:
`foreman_proxy_tftp_server` sounds good to me. :)